tests/unit/configuration_subsystem/test_navigator_post_processor.py

```diff
@@ -0,0 +1,23 @@
+"""Tests for the navigator config post-processor."""
+
+import pytest
+
+from ansible_navigator.configuration_subsystem.navigator_post_processor import NavigatorVolumeMount
+
+
+@pytest.mark.parametrize(
+    ("volmount", "expected"),
+    (
+        (NavigatorVolumeMount("/foo", "/bar"), "/foo:/bar"),
+        (NavigatorVolumeMount("/foo", "/bar", "z"), "/foo:/bar:z"),
```

Comment:
https://github.com/ansible/ansible-navigator/discussions/757

just to keep track of it, nothing to do with this PR, and it makes the review hard because there is _so_ many of these

Reply:
Unfortunately, the existence of black stands in a way of implementing this and it also makes it harder to send suggested changes via GitHub with the quote variant that is harder to type (`Shift + '`).